.github/dockerfiles/Dockerfile.build_test

```
@@ -3,9 +3,10 @@ ARG BASE
 FROM ${BASE}
 
 ARG PRE
-ENV PRE=${PRE}
 
-COPY . /netdata
+RUN /bin/sh -c 'if [ -n "${PRE}" ]; then /bin/sh -c "${PRE}"; if [ $? -ne 0 ]; then printf "ERROR: PRE failed:\n%s\n" "${PRE}"; exit 1; fi; fi'
```

Is the outer '/bin/sh -c' needed?

```suggestion
RUN if [ -n "${PRE}" ]; then /bin/sh -c "${PRE}"; if [ $? -ne 0 ]; then printf "ERROR: PRE failed:\n%s\n" "${PRE}"; exit 1; fi; fi
```

I might even write it as:

```suggestion
RUN if [ -n "${PRE}" ]; then /bin/sh -c "${PRE}" || { printf "ERROR: PRE failed:\n%s\n" "${PRE}"; exit 1; }; fi
```

for readability.

---

Maybe not I'll check :) Depends on how `RUN` are `exec`'d :)